static/css/translations.css

```diff
@@ -0,0 +1,7 @@
+td,th {
+  vertical-align: top;
+  text-align: left;
+  border: 1px solid
+}
+table {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Rule is empty.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1336404866/issues/source?bid=2452379&fileBranchId=2957759#l6)